hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/HapiPropertySource.java

```diff
@@ -245,6 +256,14 @@ static String asTopicString(TopicID topic) {
         return String.format(ENTITY_STRING, topic.getShardNum(), topic.getRealmNum(), topic.getTopicNum());
     }
 
+    static ServiceEndpoint asServiceEndpoint(String v) {
+        String[] parts = v.split(":");
+        return ServiceEndpoint.newBuilder()
+                .setIpAddressV4(ByteString.copyFromUtf8(parts[0]))
```

Comment:
v could be domain name, then need to use setDomainName()

Reply:
This is a good catch and needs to be fixed. However, the focus of this PR is on getting just a basic "works once" hapi test running. A future PR will need to add a lot more tests and fix problems such as this one.

Reply:
Ok, I added //todo to my notes.